src/components/cardProjectDetail/CardProjectDetail.tsx

```diff
@@ -254,6 +261,19 @@ const CardProjectDetail: FC<Props> = ({
             isNameBold
           />
         </div>
+        {(status === 'APPROVED' || status === 'COMPLETED') && (
```

Comment:
Instead of this strings always try to import status from status.ts file in toolkit constants